.github/workflows/build-image.yml

```diff
@@ -47,10 +47,27 @@ jobs:
       #     path: ./container-cache-out
       #     key: layers-${{ runner.os }}-${{ github.ref }}-${{ hashFiles('Dockerfile') }}
 
+      - name: Build podman container image
+        uses: ./.github/actions/image-build
+        with:
+          path: .
+          dockerfile: ./Dockerfile
+          build_target: workflow-engine-podman
+          workflow_engine_name: workflow-engine-podman-local
+          image_tag: ghcr.io/cms-enterprise/batcave/workflow-engine:podman-latest
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Scan container image
         uses: ./.github/actions/image-scan
         with:
-          image_tag: ghcr.io/cms-enterprise/batcave/workflow-engine:v0.0.1-rc.4
+          image_tag: ghcr.io/cms-enterprise/batcave/workflow-engine:latest
```

Comment:
We don't want to open the "latest" can of worms. Use a commit hash

Reply:
Okay.  I updated it to use a short commit sha.